spec/std/yaml/from_yaml_spec.cr

```diff
@@ -0,0 +1,397 @@
+require "spec"
+require "yaml"
+
+private class ObjectStub
+  def self.new(pp : YAML::PullParser)
+    pp.read_scalar.should eq "hello"
```

Comment:
I think it's bad design to have the actual test in a stub object it should be directly in the `it` spec.